src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c

```diff
@@ -2628,7 +2631,7 @@ void func_80904D88(BossGanon2* this, PlayState* play) {
 
         for (i = 0; i < 15; i++) {
             Matrix_Translate(this->unk_234[i].x, this->unk_234[i].y, this->unk_234[i].z, MTXMODE_NEW);
-            Matrix_ReplaceRotation(&play->billboardMtxF);
+            Matrix_ReplaceRotation(&play2->billboardMtxF);
```

Comment:
generic Actor* and Gamestate* theory doesnt quite make sense for this anyway since its not a main function. Its ofc possible that they purposefully made some functions take the base state as well, but it'd certainly be weird. and yeah the usage here doesnt quite make sense anyway

might be worth a comment to revisit for a different match sometime

Reply:
I changed play to `GameState* gameState` in all these "draw helper functions" (I think) and it matched and got rid of a bunch of padding variables. What do you think?

Reply:
yeah these look good to me, is probably what it was